xmtp_mls/src/subscriptions.rs

```diff
@@ -75,7 +75,20 @@ where
             .await;
 
         if let Some(err) = creation_result.as_ref().err() {
-            return Err(ClientError::Generic(err.to_string()));
+            let conn = self.context.store.conn()?;
+            let result = conn.find_group_by_welcome_id(welcome_v1.id as i64);
+            match result {
+                Ok(Some(group)) => {
+                    println!("Found existing group by welcome id");
```

Comment:
Should this be log::info!()? Might make sense to also include either the group_id or welcome_id for debugging

Reply:
added welcome id here 6dc2001e68985897e2bba1e838d4ad379d856051